documentation/4.0/content/managing-domains/domain-resource.md

```diff
@@ -206,8 +206,15 @@ The elements `serverStartPolicy`, `serverPod` and `serverService` are repeated u
 Elements related to the customization of liveness and readiness probes:
 * See [Liveness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#liveness-probe-customization" >}}) for details about the elements related to liveness probe customization and [Readiness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#readiness-probe-customization" >}}) for details about the elements related to readiness probe customization.
 
+Cluster spec elements:
```

Comment:
Instead of "Cluster spec elements:" let's try to make the intro sentence line up with the one for the Domain spec elements, maybe something along these lines (feel free to edit this intro sentence in any way that you see fit): 

The Cluster spec section contains elements for configuring the lifecycle options for all of the Managed Server members of a WebLogic cluster, including Java options, environment variables, additional Pod content, and the ability to explicitly start, stop, or restart cluster members. For a complete list of the Cluster spec elements, see [Cluster Spec](https://github.com/oracle/weblogic-kubernetes-operator/blob/{{< latestMinorVersion >}}/documentation/domains/Cluster.md#cluster-spec).

Reply:
Changed in [29738e8](https://github.com/oracle/weblogic-kubernetes-operator/pull/3662/commits/29738e819adaaee14b63d41524d7421c4dfdb07a). Let me know if this is not what you meant.

Reply:
@ankedia It isn't exactly what I meant but it can work with some modifications. I can discuss them with you on Slack and incorporate them if you agree.